src/test/java/com/github/susom/database/test/CommonTest.java

```diff
@@ -32,25 +32,14 @@
 import java.util.List;
 import java.util.TimeZone;
 
+import com.github.susom.database.*;
```

Comment:
Might check you IDE settings - we generally don't use * except for static imports

Reply:
OK - I will look at that and reset back to just what we need.